### DISCUSSION TITLE
I want to translate Korean

### DISCUSSION THREAD

**Discussion by toybrand** [Acly/krita-ai-diffusion#1051] [General]
Where is the file to be translated?

**Comment by Acly** [comment]
This one: https://github.com/Acly/krita-ai-diffusion/blob/main/ai_diffusion/language/new_language.json.template

See also instructions/guide here: https://github.com/Acly/krita-ai-diffusion/blob/main/CONTRIBUTING.md#translations-localization

**Comment by toybrand** [comment]
The translation has been completed and attached
Please let me know if there is a problem with the translation. I will
correct it
And is there a way to run Marie translation?
Even if you put it in the language folder, there is no Korean in the
language settings list
I don't know what's wrong, if there's any other way, I'd appreciate it if
you could let me know.

Hello, I'm the animation director

Thank you for making a good program
To revitalize KritaAI in Korea
I'm preparing a KritaAI lecture for students
I'd like to ask you something.
[image: screenshot-2.webp]
1. I'd like to know what model you used to shoot this video
2. I'd like to get a recommendation for an animated character model that
works well on Krita AI
3. Do you happen to have a video or lecture on the masks used in KritaAi?
If you have one, please let me know the link address

------------------------------
*보낸 사람:* Acly ***@***.***>
*보낸 날짜:* 2024년 8월 10일 토요일 오전 1:20
*받는 사람:* Acly/krita-ai-diffusion ***@***.***>
*참조:* toybrand ***@***.***>; Author ***@***.***>
*제목:* Re: [Acly/krita-ai-diffusion] I want to translate Korean (Discussion
#1051)


This one:
https://github.com/Acly/krita-ai-diffusion/blob/main/ai_diffusion/language/new_language.json.template

See also instructions/guide here:
https://github.com/Acly/krita-ai-diffusion/blob/main/CONTRIBUTING.md#translations-localization

—
Reply to this email directly, view it on GitHub
<https://github.com/Acly/krita-ai-diffusion/discussions/1051#discussioncomment-10289275>,
or unsubscribe
